k8s.io/canary.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function kc() {
```

Comment:
aliases are not expanded by bash in non-interactive shells.  I tried it :)

On Tue, Aug 23, 2016 at 1:56 PM, Ryan Hitchman <notifications@github.com>
wrote:

> In k8s.io/canary.sh
> <https://github.com/kubernetes/k8s.io/pull/4#discussion_r75949011>:
>
> > @@ -0,0 +1,31 @@
> > +#!/bin/bash
> > +
> > +set -o errexit
> > +set -o nounset
> > +set -o pipefail
> > +
> > +function kc() {
>
> this could be an alias instead of a function
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/k8s.io/pull/4/files/ddaa006766e2fcac13ea61f08541296af5c537c9#r75949011>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFVgVL6qUEceIwvMbbOvMkNe36BjZLl-ks5qi15igaJpZM4JqiuI>
> .
>